src/Symfony/Component/HttpKernel/DataCollector/RequestDataCollector.php

```diff
@@ -63,6 +64,10 @@ public function collect(Request $request, Response $response, \Exception $except
             if ('_route' === $key) {
                 $route = is_object($value) ? $value->getPath() : $value;
             }
+
+            if ('_route_params' === $key && is_array($value)) {
+                $routeParams = $value;
```

Comment:
@nicolas-grekas now the request attributes break :sob: 

![image](https://cloud.githubusercontent.com/assets/1047696/20496338/0f8d16da-b025-11e6-9033-79a4144703fd.png)

Not sure we can support this with only 1 structural value..